mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -823,6 +825,12 @@ class WidgetAdapter(
             } else {
                 group.check(checkedId)
             }
+
+            // Layout has to be drawn before measuring the width
+            handler.post {
```

Comment:
There's zero guarantee for the layout to be finished in the next main loop iteration. This needs to be solved by using a layout listener.

Reply:
I'll check that.